fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeChecker.java

```diff
@@ -977,6 +980,11 @@ public Void visitRecoverDbStatement(RecoverDbStmt statement, ConnectContext sess
         @Override
         public Void visitShowFunctionsStatement(ShowFunctionsStmt statement, ConnectContext context) {
             String dbName = statement.getDbName();
+            // No need to check privilege when `show global functions`
+            if (statement.getIsGlobal()) {
```

Comment:
also need to check at createFunctionStatement and dropFunctionStatement?

Reply:
I guess no need to, because right now to create DB UDF , user needs permisson of ADMIN of global level. It's same to GLOBAL UDF. 

```
        @Override
        public Void visitDropFunctionStatement(DropFunctionStmt statement, ConnectContext context) {
            // check operation privilege
            if (!GlobalStateMgr.getCurrentState().getAuth()
                    .checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
                ErrorReport.reportSemanticException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
            }
            return null;
        }

        @Override
        public Void visitCreateFunctionStatement(CreateFunctionStmt statement, ConnectContext context) {
            // check operation privilege
            if (!GlobalStateMgr.getCurrentState().getAuth()
                    .checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
                ErrorReport.reportSemanticException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
            }
            return null;
        }
```

Reply:
Yes,  it's a little different. Global UDF needs CREATE/DROP of global level.